docs/standard/technical/deployment.rst

```diff
@@ -75,6 +75,11 @@ Before each release, and at least once a year:
 
    python manage.py update-country --help
 
+-  check `ISO6523 <https://www.iso.org/standard/25773.html>`__ for new organization identifier schemes:
+
+   - Compare the `PEPPOL BIS Billing 3.0 ICD codelist <https://docs.peppol.eu/poacc/billing/3.0/codelist/ICD/>`__ to the `ISO6523-analysis spreadsheet <https://docs.google.com/spreadsheets/d/1qPPAOx9w4NIfrll66a0Qd2SMizoxGNRUMU6YV2rLuEE/edit#gid=1900375069>`__ to identify new codes.
```

Comment:
Do we use PEPPOL because there is no better or more direct source for the ISO 6523 codelist?

Reply:
I added a command to do the comparison: https://github.com/open-contracting/standard/pull/1501

Reply:
Also, I don't have edit access to this sheet, so I can't add codes.

Reply:
> Do we use PEPPOL because there is no better or more direct source for the ISO 6523 codelist?

Yes, that's right.

> I added a command to do the comparison: https://github.com/open-contracting/standard/pull/1501

Looks good!

> Also, I don't have edit access to this sheet, so I can't add codes.

I've requested permission to move the spreadsheet to the helpdesk folder

Reply:
I'm not sure what account receives the permission request.

Please update the instructions to use the command. I've added a comment about the use of PEPPOL in the script.

Reply:
> I'm not sure what account receives the permission request.

It's currently filed in an ODS shared drive to which I don't have manager permissions so I've asked a colleague to grant me permission.

> Please update the instructions to use the command. I've added a comment about the use of PEPPOL in the script.

Done in https://github.com/open-contracting/standard-development-handbook/pull/270/commits/f32fd915524a4b3c7157ba7dae5f97aeaa4bccb5

Reply:
@jpmckinney you should now have edit access to the spreadsheet, along with anyone else who has access to the Helpdesk folder on Google Drive.